core-web/libs/sdk/experiments/src/lib/hooks/DotExperimentsContext.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext } from 'react';
+
+import { DotExperiments } from '../dot-experiments';
+
+/**
+ * `DotExperimentsContext` is a React context that is designed to provide an instance of
+ * `DotExperiments` to all of the components within its tree that are Consumers of this context.
+ *
+ * The context is created with a default value of `null`. It is meant to be provided a real value
+ * using the `DotExperimentsProvider` component.
+ *
+ * @example
+ * ```jsx
+ * // import the context
+ * import DotExperimentsContext from './DotExperimentsContext';
+ *
+ * // Use in function component
+ * function MyComponent() {
+ *   const dotExperimentsInstance = React.useContext(DotExperimentsContext);
+ *   // Use the dotExperimentsInstance here...
+ * }
+ *
+ * // Or with Class component
+ * class MyClassComponent extends React.Component {
+ *   static contextType = DotExperimentsContext;
+ *   // The instance is now accessible as this.context
+ * }
```

Comment:
I don't think we need to add class component examples.

Reply:
not really. Removed.